invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -78,11 +75,55 @@ def requests_detail(request=None, pid_value=None):
             default_query_config=default_query_config
         )
 
-    except TemplateNotFound:
-        return render_template(
-            "invenio_requests/details/index.html",
-            # TODO: implement and use request UI serializer
-            request=request_dict,
-            record=topic_record,
-            default_query_config=default_query_config
-        )
+
+def invitation_details(community_pid_value=None,
```

Comment:
Shouldn't this require the user to login?

Reply:
it still has the `@login_required decorator` :smile:

Reply:
I see it in the `requests_detail` view but not on this one, no? If so, maybe you can move it to the `requests_view` one

Reply:
ah my bad, you are right.